src/dto/create-game-request.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsBoolean, IsNotEmpty } from "class-validator";
+
+export class CreateGameRequestDto {
+    @IsNotEmpty()
+    @IsBoolean()
+    isTimeBasedScore: boolean;
+
+    @IsNotEmpty()
+    @IsBoolean()
+    isPunishmentScoreAllowed: boolean;
+
+    @IsNotEmpty()
+    @IsBoolean()
+    allowBuzzerTwice: boolean;
```

Comment:
use the same naming convention. `isBuzzerTwiceAllowed`

Reply:
d018182d058bc9049e8133ab1f20f7588995fa7c